python/private/pypi/pep508_platform.bzl

```diff
@@ -0,0 +1,57 @@
+# Copyright 2025 The Bazel Authors. All rights reserved.
```

Comment:
FYI, going forward, we can omit these copyright headers

Reply:
Should we do with the old ones?